go-controller/pkg/ovn/namespace.go

```diff
@@ -112,21 +111,21 @@ func (oc *Controller) deletePodFromNamespace(ns string, portInfo *lpInfo) error
 // Creates an explicit "allow" policy for multicast traffic within the
 // namespace if multicast is enabled. Otherwise, removes the "allow" policy.
 // Traffic will be dropped by the default multicast deny ACL.
```

Comment:
minor suggestion: add a comment that the `nsInfo` lock should be held.

Reply:
No code outside of the oc.namespaces-managing code should ever hold an unlocked `namespaceInfo`. I clarified the `namespaceInfo` and `oc.namespaces` docs a little more